code/ui/manager/src/components/mobile/navigation/MobileMenuDrawer.tsx

```diff
@@ -0,0 +1,130 @@
+import type { FC } from 'react';
+import React, { useRef } from 'react';
+import { styled } from '@storybook/theming';
+import { Transition } from 'react-transition-group';
+import type { TransitionStatus } from 'react-transition-group/Transition';
+import { MobileAbout } from '../about/MobileAbout';
+import Sidebar from '../../../container/Sidebar';
+import { MOBILE_TRANSITION_DURATION } from '../../../constants';
+
+interface MobileMenuDrawerProps {
+  isMenuOpen: boolean;
+  isAboutOpen: boolean;
+  setAboutOpen: (open: boolean) => void;
+  closeMenu: () => void;
+}
+
+export const MobileMenuDrawer: FC<MobileMenuDrawerProps> = ({
+  isMenuOpen,
+  isAboutOpen,
+  setAboutOpen,
+  closeMenu,
+}) => {
+  const containerRef = useRef(null);
+  const sidebarRef = useRef(null);
+  const overlayRef = useRef(null);
+
+  return (
+    <>
+      <Transition
+        nodeRef={containerRef}
+        in={isMenuOpen}
+        timeout={MOBILE_TRANSITION_DURATION}
+        mountOnEnter
+        unmountOnExit
+      >
+        {(state) => (
+          <Container ref={containerRef} state={state}>
+            <Transition nodeRef={sidebarRef} in={!isAboutOpen} timeout={MOBILE_TRANSITION_DURATION}>
+              {(sidebarState) => (
+                <SidebarContainer ref={sidebarRef} state={sidebarState}>
+                  <Sidebar setAboutOpen={setAboutOpen} />
```

Comment:
In the Sidebar components and all it's children, this should be called `onMenuClick`, so the Sidebar doesn't need to know anything about an about page that is open or closed. it just fires the onclick, and then the drawer here will know to use that to open/close the menu